### ISSUE TITLE
import/export process doesn't seem to export the images for the custom fields

### ISSUE THREAD

**Issue by yochannah** [kushinauwu/OBF-Homepage-New#10]
let's try using https://wordpress.org/plugins/all-in-one-wp-migration/#reviews to see if this fixes it.

This may impact #2 !

**Comment by kushinauwu** [comment]
Used this plugin, and it seems fine on my end, but I'm not sure if it is alright when the whole site is imported on a remote location like your machine. I didn't have to make a new menu this way though, so that means even the menu got imported, so this might just work!

**Comment by yochannah** [comment]
I've given this a go - the level of detail in the instructions is perfect! I've tried it on an empty / brand new wp and it mostly works great, but I encountered a problem at the theme installation steps 3 / 4 - it deletes my user when it imports the new theme, and then I can't log in! 😆 your user is the only user in the db (the password is hashed so I can't see it though). 

The simplest workaround might be to make a user "yo" with password yo and include that in your export file?
